app/src/main/java/com/example/tasky/agenda/domain/GetInitialsUseCase.kt

```diff
@@ -0,0 +1,22 @@
+package com.example.tasky.agenda.domain
+
+import com.example.tasky.authentication.domain.UserCache
+import javax.inject.Inject
+
+class GetInitialsUseCase @Inject constructor(
+    private val userCache: UserCache
```

Comment:
You could do this...

Or even better, you can pass the name you want the initials for, in the invoke parameter
This makes it more versatile, since you will be able to accept other names, not only the one stored in the SharedPreference